.github/workflows/tests.yaml

```
@@ -16,7 +16,7 @@ jobs:
         run: sudo apt-get install -y build-essential
 
       - name: setup python 3
-        uses: actions/setup-python@dc73133d4da04e56a135ae2246682783cc7c7cb6 # pin@v2
+        uses: actions/setup-python@v4 # pin@v2
```

```suggestion
        uses: actions/setup-python@v4
```

Let's also remove the comment?